Makefile

```diff
@@ -1,6 +1,29 @@
+UNAME_S ?= $(shell uname -s)
+UNAME_M ?= $(shell uname -m)
+FFMPEG_PREFIX ?= $(shell pwd)/ffmpeg/$(UNAME_S)-$(UNAME_M)
+FFMPEG_OPTS ?= --prefix=$(FFMPEG_PREFIX) \
+               --enable-static \
+               --disable-shared \
+               --disable-programs \
+               --disable-doc \
+               --disable-everything \
+               --enable-decoder=h264 \
+               --enable-decoder=hevc \
+               --enable-swscale
+CGO_LDFLAGS := -L$(FFMPEG_PREFIX)/lib
+ifeq ($(UNAME_S),Linux)
+ifneq ($(shell find / -name libjpeg.a 2> /dev/null),)
+	CGO_LDFLAGS := "$(CGO_LDFLAGS) -l:libjpeg.a"
+endif
+endif
 
-bin/viamrtsp: *.go cmd/module/*.go
-	go build -o bin/viamrtsp cmd/module/cmd.go
+.PHONY: build-ffmpeg test lint updaterdk module
+
+bin/viamrtsp: build-ffmpeg *.go cmd/module/*.go
+	PKG_CONFIG_PATH=$(FFMPEG_PREFIX)/lib/pkgconfig \
+		CGO_CFLAGS=-I$(FFMPEG_PREFIX)/include \
+		CGO_LDFLAGS="-L$(FFMPEG_PREFIX)/lib -l:libjpeg.a" \
```

Comment:
Isn't this going to overwrite `CGO_LDFLAGS` here: https://github.com/erh/viamrtsp/pull/5/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R13 and here https://github.com/erh/viamrtsp/pull/5/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R16

?